Maddori.Apple/Maddori.Apple/Global/Literal/TextLiteral.swift

```diff
@@ -234,4 +234,16 @@ enum TextLiteral {
     // MARK: - TeamDetailViewController
     
     static let teamDetailViewControllerEditButteonText: String = "편집"
+    
+    // MARK: - TeamManageViewController
+    
+    static let teamManageViewControllerEmptyLabel: String = "아직 팀에 합류한 팀원이 없습니다\n초대코드를 공유하여 팀원을 초대해주세요"
```

Comment:
p1;
```suggestion
    static let teamManageViewControllerEmptyLabel: String = "참여 중인 팀이 없습니다\n팀에 합류하거나 팀을 생성해주세요"
```
리터럴 수정이 필요해 보입니다!

Reply:
[a4b24a8](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/a4b24a88f77b54899e0c1586dad85b80f3c0ae93)

수정 완료!